### PR TITLE
bpfd: Add bpffs helpers to StorageManager

### PR DESCRIPTION
**Pull request by dave-tucker**
Pulls in https://github.com/bpfd-dev/aya/commit/27df828e2f95799ecf1f977a003477bbb1b2c011

### CODE REVIEW

File: bpfd/src/storage.rs [hunk 1]
```diff
@@ -203,4 +206,25 @@ impl StorageManager {
...
+    fn create_bpffs(&self, path: &Path) -> anyhow::Result<()> {
+        create_bpffs(
+            path.as_os_str()
+                .to_str()
+                .expect("unable to convert path to str"),
+        )
+    }
+
+    #[allow(dead_code)] // TODO: Remove this when the storage manager is fully implemented
+    fn pin_map_to_bpffs(
```
Comment (astoycos) [inline bpfd/src/storage.rs:220]: So for duping a map to a new bpffs we'll just do an aya `from_pin` call to get the mapdata and then pin again to the new bpffs right?

Comment (dave-tucker) [reply bpfd/src/storage.rs:220]: Yep. The whole process is:

- MapData::from_pin() - which behind the scenes does BPF_OBJ_GET from $basePath
- Then MapData::pin() - which does BPF_OBJ_PIN to $targetPath

On my TODO list is to check whether we can `cp` the pinned object from one bpffs to another.
That's riskier, so we'll stick with the above for now.

Comment (astoycos) [reply bpfd/src/storage.rs:220]: Sounds good to me Thanks!

**Review by astoycos — APPROVED** [review]
LGTM

Thanks @dave-tucker

**Pull request merged**